part-time/code/4220-stefan-milkovski/src/app/recipe-list/recipe-list.component.ts

```diff
@@ -7,4 +8,12 @@ import { Component } from '@angular/core';
 })
 export class RecipeListComponent {
 
+  constructor(private http: HttpClient) {}
+  
+  private recipeListApi = "https://localhost:2999";
+
+  getRecipeApi() {
```

Comment:
The data should be loaded via a service, not directly from the component. Anyway, this method is not even called and used.